packages/app/src/LogSidePanel.tsx

```diff
@@ -2455,12 +2460,30 @@ const MetricsSubpanel = ({ logData }: { logData?: any }) => {
   return (
     <Stack my="md" spacing={40}>
       {podUid && (
-        <MetricsSubpanelGroup
-          title="Pod Metrics"
-          where={`k8s.pod.uid:"${podUid}"`}
-          fieldPrefix="k8s.pod."
-          timestamp={timestamp}
-        />
+        <div>
+          <MetricsSubpanelGroup
+            title="Pod Metrics"
+            where={`k8s.pod.uid:"${podUid}"`}
+            fieldPrefix="k8s.pod."
+            timestamp={timestamp}
+          />
+          <Card p="md" mt="xl">
+            <Card.Section p="md" py="xs" withBorder>
+              Pod Timeline
+            </Card.Section>
+            <Card.Section>
+              <ScrollArea
+                viewportProps={{
+                  style: { maxHeight: 280 },
+                }}
+              >
+                <Box p="md" py="sm">
+                  <KubeTimeline q={`k8s.pod.uid:"${podUid}"`} />
```

Comment:
would be nice to fetch only `event +- time` and show event in the timeline

Reply:
added it
![Screenshot 2024-01-10 at 8 56 28 PM](https://github.com/hyperdxio/hyperdx/assets/149748269/5dc3efb0-d2dc-4560-8dda-a9d1c578e82c)